source/broadband/webpa_notification.c

```diff
@@ -282,6 +291,98 @@ void FactoryResetCloudSyncTask()
 		WalInfo("FactoryResetCloudSync Thread created Successfully\n");
 	}
 }
+void *SyncNotifyRetry()
+{
+	pthread_detach(pthread_self());
+	char *dbCMC = NULL;
+	int retryCount = 0;
+	int backoffRetryTime = 0;
+	int backoff_max_time = 9;
+	struct timespec ts;
+	int c = 3;
+	int  rv;    
+	int max_retry_sleep = (int) pow(2, backoff_max_time) -1;
+	WalInfo("max_retry_sleep is %d\n", max_retry_sleep );
```

Comment:
Add func name also here as we have this print at other places also.

WalInfo("SyncNotifyRetry max_retry_sleep is %d\n", max_retry_sleep );